app/views/saved_scenarios/show.html.haml

```diff
@@ -4,12 +4,16 @@
     .warning
       = @warning
 
-  %h1= @scenario.title.nil? ? t('scenario.no_title').truncate(40, separator: ' ') : @scenario.title.truncate(40, separator: ' ')
+  %h1= (@scenario.title or t 'scenario.no_title' ).truncate(40, separator: ' ')
 
-  = link_to t("scenario.load"), load_saved_scenario_path(@saved_scenario), class: "button primary"
+  = link_to t("scenario.load"),
+            load_saved_scenario_path(@saved_scenario),
+            class: "button primary"
+
+  = link_to t("scenario.load_factsheet"),
+            factsheet_scenario_path(@saved_scenario.scenario_id),
+            class: "button primary"
```

Comment:
Could we perhaps remove the `primary` class from "Open infographic" so that the "Open this scenario" button stands out more?

Reply:
good idea